### PR TITLE
Update Codeanywhere

### PR DESCRIPTION
**Pull request by arcuslinus**
Codeanywhere changed their tfa-system.

### CODE REVIEW

**Comment by Carlgo11** [comment]
Could you provide any documentation pages confirming this?

**Comment by arcuslinus** [comment]
Unfortunately they haven't updated their docs yet, but I can attach a screenshot:

image

File: _data/developer.yml [hunk 1]
```diff
@@ -137,8 +137,7 @@ websites:
       url: https://codeanywhere.com
       img: codeanywhere.png
       tfa: Yes
-      sms: Yes
-      doc: https://docs.codeanywhere.com/features/twofactorauth.html
```
Comment (mxxcon) [inline _data/developer.yml:141]: Why are you deleting docs url? Is not getting removed?

Comment (arcuslinus) [reply _data/developer.yml:141]: They are outdated, which would be confusing.

**Review by Carlgo11 — APPROVED** [review]

**Review by stephengroat — APPROVED** [review]

**Pull request merged**